MentorBook.Business/Interfaces/IFriendshipService.cs

```diff
@@ -7,5 +7,7 @@ namespace MentorBook.Business
     public interface IFriendshipService
     { 
         int GetUserFriendsCountById(int UserId);
+        List<Friend> GetFriendById(int id);
+        List<Friend> GetPendingFriend(int id);
```

Comment:
The name should be plural - GetPendingFriends as you return collection of multiple ones.